MaxText/configs/base.yml

```diff
@@ -297,6 +297,14 @@ logical_axis_rules: [
                       ['embed_no_exp', ['fsdp', 'sequence', 'tensor_transpose']],
                       ['embed_no_exp', ['fsdp', 'fsdp_transpose', 'sequence']],
                       ['embed_no_exp', ['fsdp', 'sequence']],
+                      ['q_lora', ['fsdp', 'fsdp_transpose', 'sequence', 'tensor_transpose', 'expert']],
+                      ['q_lora', ['fsdp', 'sequence', 'tensor_transpose', 'expert']],
+                      ['q_lora', ['fsdp', 'fsdp_transpose', 'sequence', 'expert']],
+                      ['q_lora', ['fsdp', 'sequence', 'expert']],
+                      ['kv_lora', ['fsdp', 'fsdp_transpose', 'sequence', 'tensor_transpose', 'expert']],
+                      ['kv_lora', ['fsdp', 'sequence', 'tensor_transpose', 'expert']],
+                      ['kv_lora', ['fsdp', 'fsdp_transpose', 'sequence', 'expert']],
+                      ['kv_lora', ['fsdp', 'sequence', 'expert']],
```

Comment:
Are you needing LoRA for inference or training? Let me know if you need it for training since there isn't MaxText support for it currently

Reply:
This is for DeepSeek support, MLA Attention has `q_lora` and `kv_lora` for latent low rank projection.